### DISCUSSION TITLE
Suggestion/Feature Request: !glory command for colosseum glory

### DISCUSSION THREAD

**Discussion by dcooke13** [runelite/runelite#17977] [Ideas]
Title. I do not believe this exists yet but would be a good feature for bragging rights.

**Comment by Felanbird** [comment]
While !glory is not yet an* official™️ command, the `!lvl` command does a hiscore lookup of whatever you've typed in, so you can do `!lvl Colosseum Glory` (exact match) and get your glory amount

**Reply by dcooke13** [reply]
Thanks!
